sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -1029,6 +1030,32 @@ class JDBCSuite extends QueryTest
     }
   }
 
+  test("Hide credentials in show create table") {
+    val password = "testPass"
```

Comment:
Can we explicitly set the conf in the test?

Reply:
Do you mean remove `password` declaration?